pkg/context/cas/client.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package cas
+
+import (
+	"sync"
+
+	"github.com/pkg/errors"
+
+	"github.com/hyperledger/fabric-sdk-go/pkg/client/channel"
+	"github.com/hyperledger/fabric-sdk-go/pkg/common/providers/context"
+)
+
+const (
+	sidetreeTxnCC = "sidetreetxn_cc"
+	writeFcn      = "writeContent"
```

Comment:
Will there be other read and write functions?

(or is it effectively going to be GET and PUT/POST).